base/deprecated.jl

```diff
@@ -1350,6 +1350,15 @@ end
 @deprecate srand(filename::AbstractString, n::Integer=4) srand(read!(filename, Array{UInt32}(Int(n))))
 @deprecate MersenneTwister(filename::AbstractString)  srand(MersenneTwister(0), read!(filename, Array{UInt32}(Int(4))))
 
+# Part of the underscore audit
+for f in [:is_apple, :is_bsd, :is_linux, :is_unix, :is_windows]
+    newf = Symbol(replace(String(f), "_", ""))
+    @eval begin
+        @deprecate $f(os) $newf(os)
+        @deprecate $f() $newf()
+    end
+end
+
```

Comment:
It works because all methods of `is_apple` etc. are deprecated, so it is not necessary to single out some of them :)